spec/reek/spec/should_reek_of_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'pathname'
 require_relative '../../spec_helper'
 require_lib 'reek/spec'
+require 'active_support/core_ext/string/strip'
+require 'active_support/core_ext/hash/except'
```

Comment:
F**k. Thanks for paying attention (because I clearly didn't), my bad, pull request to fix this incoming.